docs/content/mixins/_index.md

```diff
@@ -1,15 +1,7 @@
 ---
 title: Mixins
 description: Available Mixins
-layout: single
 ---
 
-Mixins make Porter special, find them, use them, [create your own](/mixin-dev-guide).
-
-* [exec](/mixins/exec)
-* [helm](/mixins/helm)
-* [kubernetes](/mixins/kubernetes)
-* [az](/mixins/az)
-* [aws](/mixins/aws)
-* [gcloud](/mixins/gcloud)
-* [terraform](/mixins/terraform)
+Mixins make Porter special, they are the building blocks that you use when
```

Comment:
nit: consider adjusting punctuation.  Perhaps `Mixins make Porter special.  They are the...` ?  Or, a semicolon in place of the comma.